wgpu-core/src/lib.rs

```diff
@@ -67,7 +67,7 @@ pub mod registry;
 pub mod resource;
 pub mod storage;
 mod track;
-mod validation;
+pub mod validation;
```

Comment:
Could you add a note why this is public and what would allow us to bring it private again?

Reply:
Done.
Continued in https://github.com/gfx-rs/wgpu/pull/4811#discussion_r1417940633.